config/environment.js

```diff
@@ -51,6 +51,7 @@ module.exports = function(environment) {
         modulePrefix: 'ember-osf-web',
         environment,
         rootURL: '/',
+        assetsPrefix: '/',
```

Comment:
Might be nice to put this in an environment variable, at least so it can be set in `config/local.js`

Reply:
That's a great idea! This would also allow this to be set in production deploys, say, if we move assets to a CDN.